pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -640,47 +639,18 @@ func (oe *operationExecutor) VerifyVolumesAreAttached(
 				continue
 			}
 
-			// Migration: Must also check the Node since Attach would have been done with in-tree if node is not using Migration
-			nu, err := nodeUsingCSIPlugin(oe.operationGenerator.GetCSITranslator(), oe.operationGenerator.GetVolumePluginMgr(), volumeAttached.VolumeSpec, node)
-			if err != nil {
-				klog.Errorf(volumeAttached.GenerateErrorDetailed("VolumesAreAttached.NodeUsingCSIPlugin failed", err).Error())
+			volumePlugin, err :=
+				oe.operationGenerator.GetVolumePluginMgr().FindPluginBySpec(volumeAttached.VolumeSpec)
+			if err != nil || volumePlugin == nil {
+				klog.Errorf(
+					"VolumesAreAttached.FindPluginBySpec failed for volume %q (spec.Name: %q) on node %q with error: %v",
```

Comment:
Is `FindPluginBySpec failed... with error: nil` a reasonable description of the problem when both `err` and `volumePlugin` are nil?

Reply:
Good point ... will fix.